include/mesh/do_nothing_inserter_iterator.h

```diff
@@ -0,0 +1,73 @@
+// The libMesh Finite Element Library.
+// Copyright (C) 2002-2022 Benjamin S. Kirk, John W. Peterson, Roy H. Stogner
+
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+
+
+
+#ifndef LIBMESH_DO_NOTHING_INSERTER_ITERATOR_H
+#define LIBMESH_DO_NOTHING_INSERTER_ITERATOR_H
+
+// C++ includes
+#include <iterator>
+
+namespace libMesh
+{
+
+/**
+ * A class for templated methods that expect output iterator
+ * arguments, but for which we want to take no action when something
+ * tries to write through the inserter. This is useful in packed_range
+ * functions in which we are (un)packing elements or nodes into/from a
+ * mesh. We specialize the packing of nodes and elements with a mesh
+ * context such that the (un)packing process adds directly into the mesh.
+ * Given such a specialization, we don't want to actually do anything
+ * with the output iterator, which motivates the creation of this class
+ *
+ * \author Alexander Lindsay
+ * \date 2022
+ * \brief A do-nothing output iterator for use with packed_range functions.
+ */
+template <typename T>
+struct do_nothing_inserter_iterator
```

Comment:
Is there something that this can do that include/utils/null_output_iterator.h can't?

Reply:
I would think not! Let me check! Clearly my searching for pre-existing options was not thorough enough

Reply:
looks like `null_output_iterator` worked like a charm